### ISSUE TITLE
API readiness milestone tasks not reflected as completed after associating new API Spec PR

### ISSUE THREAD

**Issue by ladonnaq** [Azure/azure-sdk-tools#9286]
**Issue:** The Service Partner updated the API Spec PR link in the API Readiness milestone. All of the checks passed, and ARM has signed off. However, the release plan summary incorrectly shows the tasks as incomplete. While this is a non-blocking issue, it causes confusion, which is why the Service Partner reached to us via the Azure SDK Onboarding channel and also attended office hours.  

Link to release plan - https://aka.ms/sdk-release-planner?release-plan-id=c8bd6837-e86a-ef11-bfe3-6045bd08e35c
Latest API Spec PR - https://github.com/Azure/azure-rest-api-specs/pull/30835

**Comment by maririos** [comment]
I looked into this and although the information that is being reported is correct, the app has the `Next` buttons of each task disabled, so there is no way for the user to `complete` each task again.
They can complete the API readiness step though but it is still confusing.

My suggestion here is that instead of fixing those buttons, we take advantage and remove those views from the API Readiness management plane app.

FYI @praveenkuttappan  :)

**Comment by ladonnaq** [comment]
> I looked into this and although the information that is being reported is correct, the app has the `Next` buttons of each task disabled, so there is no way for the user to `complete` each task again. They can complete the API readiness step though but it is still confusing.
> 
> My suggestion here is that instead of fixing those buttons, we take advantage and remove those views from the API Readiness management plane app.
> 
> FYI [@praveenkuttappan](https://github.com/praveenkuttappan) :)

I agree as long as all of the required prerequisites are validated before the PR merges, then we should be able to check the state of the PR (merged or not merged). Then display any relevant information.

**Comment by maririos** [comment]
This issue will go away with the work on https://github.com/Azure/azure-sdk-tools/issues/8063
